### ISSUE TITLE
[Error] Unable to process PlayerInteract event.

### ISSUE THREAD

**Issue by Mcayear** [tim03we/FuturePlots#48]
### Issue description

- **Players standing on the road can destroy other players' plots**

### Version and Provider

* FuturePlots: [FuturePlots-3.3.3-SNAPSHOT.jar](https://github.com/tim03we/FuturePlots/releases/tag/3.3.3-SNAPSHOT)
* Provider: [PowerNukkitX .19.80-r3](https://github.com/PowerNukkitX/PowerNukkitX/releases/tag/1.19.80-r3)

### Steps before you create the issue 

- [ ] If you remove all other plugins, does the issue still occur?
- [x] Have you ever tried the bug as an OP and as a non-OP player?

### Crashdump, backtrace or other files
```
08:22:42 [ERROR]  Unable to process event "cn.nukkit.event.player.PlayerInteractEvent" 至 "FuturePlots v3.3.3-SNAPSHOT"：Cannot invoke "tim03we.futureplots.utils.Plot.getLevelName()" because "plot" is null 于 tim03we.futureplots.listener.PlayerInteract
java.lang.NullPointerException: Cannot invoke "tim03we.futureplots.utils.Plot.getLevelName()" because "plot" is null
        at tim03we.futureplots.provider.data.YamlProvider.getOwner(YamlProvider.java:144) ~[FuturePlots-3.3.3-SNAPSHOT.jar:?]
        at tim03we.futureplots.provider.data.YamlProvider.hasOwner(YamlProvider.java:134) ~[FuturePlots-3.3.3-SNAPSHOT.jar:?]
        at tim03we.futureplots.utils.PlotPlayer.canByPass(PlotPlayer.java:54) ~[FuturePlots-3.3.3-SNAPSHOT.jar:?]
        at tim03we.futureplots.utils.Plot.canByPass(Plot.java:75) ~[FuturePlots-3.3.3-SNAPSHOT.jar:?]
        at tim03we.futureplots.listener.PlayerInteract.onInteract(PlayerInteract.java:46) ~[FuturePlots-3.3.3-SNAPSHOT.jar:?]
        at cn.nukkit.plugin.PNXMethodEventExecutor$105.execute(EventHandler@tim03we.futureplots.listener.PlayerInteract#onInteract) ~[?:?]
        at cn.nukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:61) ~[PowerNukkitX.jar:?]
        at cn.nukkit.plugin.PluginManager.callEvent(PluginManager.java:580) [PowerNukkitX.jar:?]
        at cn.nukkit.network.process.processor.InventoryTransactionProcessor.handleUseItem(InventoryTransactionProcessor.java:611) [PowerNukkitX.jar:?]
        at cn.nukkit.network.process.processor.InventoryTransactionProcessor.handle(InventoryTransactionProcessor.java:57) [PowerNukkitX.jar:?]
        at cn.nukkit.network.process.processor.InventoryTransactionProcessor.handle(InventoryTransactionProcessor.java:46) [PowerNukkitX.jar:?]
        at cn.nukkit.network.process.DataPacketManager.processPacket(DataPacketManager.java:47) [PowerNukkitX.jar:?]
        at cn.nukkit.Player.handleDataPacket(Player.java:3468) [PowerNukkitX.jar:?]
        at cn.nukkit.network.session.RakNetPlayerSession.serverTick(RakNetPlayerSession.java:231) [PowerNukkitX.jar:?]
        at cn.nukkit.network.RakNetInterface.process(RakNetInterface.java:96) [PowerNukkitX.jar:?]
        at cn.nukkit.network.Network.processInterfaces(Network.java:239) [PowerNukkitX.jar:?]
        at cn.nukkit.Server.tick(Server.java:1278) [PowerNukkitX.jar:?]
        at cn.nukkit.Server.tickProcessor(Server.java:1152) [PowerNukkitX.jar:?]
        at cn.nukkit.Server.start(Server.java:1133) [PowerNukkitX.jar:?]
        at cn.nukkit.Server.<init>(Server.java:960) [PowerNukkitX.jar:?]
        at cn.nukkit.Nukkit.main(Nukkit.java:160) [PowerNukkitX.jar:?]
```

**Comment by tim03we** [comment]
@Mcayear I will have a look as soon as I get home.

**Comment by Dodspa** [comment]
Hello! I’ve previously reported this bug a couple months ago and it has been fixed in the latest SNAPSHOT version (3.3.4)

**Comment by tim03we** [comment]
right there was once what have even no longer thought of the last version
